COMPARED_WITH_PRY.md

```diff
@@ -4,19 +4,19 @@
+| Document browsing         | `ri`                                                                                                                 | `show_doc`                                                                                                                                             |                                                                                                |
+| Live help system          | `help` or `command_name --help`                                                                                      | `help`                                                                                                                                                 | IRB doesn't support detailed descriptions for individual commands yet                          |
+| Open methods in editors   | `edit`                                                                                                               | `edit`                                                                                                                                                 |                                                                                                |
+| Syntax highlighting       | Yes                                                                                                                  | Yes                                                                                                                                                    |                                                                                                |
+| Command shell integration | Yes                                                                                                                  | No                                                                                                                                                     | Currently, there's no plan to support such features in IRB                                     |
+| Navigation                | - `cd object` to enter `object` <br/> - `cd ..` to leave the current object <br/> - `nesting` to list nesting levels | - `pushws object` <br/> - `popws` <br/> - `workspaces` <br /> - `cd object`/`cd ..` (since [1.14.0](https://github.com/ruby/irb/releases/tag/v1.14.0)) | We plan to refine IRB's commands in the future                                                 |
+| Runtime invocation        | `binding.pry`                                                                                                        | 👋 IRB is on a mission to match feature parity with Pry. This document is here to help us track the similarities and differences between the two.      |                                                                                                |
```

Comment:
I think from here and below the table is mixed with other things?

Reply:
Ouch. It's so brutal to me to edit MD tables. 😖 Sorry and thanks. That's fixed.